media-sound/mpd/files/mpd-0.20.4.logrotate

```diff
@@ -1,6 +1,6 @@
 /var/lib/mpd/log {
 	missingok
 	postrotate
-	/etc/init.d/mpd reload
+		/etc/init.d/mpd --quiet reload
```

Comment:
Changing this line means changing an installed file for stable 0.20.10. Rather than that, we'll copy this to a new .logrotate version on merge.